src/components/BoardView/index.js

```diff
@@ -0,0 +1 @@
+export {default} from './BoardView';
```

Comment:
Why those odd index.js files?

Reply:
Because of that I can make shorter imports of these components.

`import CardView from "../CardView";`

instead of:

`import CardView from "../CardView/CardView";`

It looks cleaner for me, even for the price of creating these files.

Reply:
So you are creating extra files to avoid "../" ? Let's not :)
A much better approach is to actually show what you are importing:
`import  "../../components/BoardView/BoardView`
+ there is no reason for this component to be in its own directory, is there?